tests/src/JIT/Regression/JitBlue/GitHub_20211/GitHub_20211.cs

```diff
@@ -0,0 +1,247 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace GitHub_20211
```

Comment:
For the purpose of validating this fix, it seems fine to test only an index of 0, but it would be useful to add a comment to this test to clarify what the bug was.

Reply:
@CarolEidt I have added a bug description to the test. PTAL

Reply:
Thanks - meant to approve before.